### PR TITLE
Delete ldap config from UI

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR...

Fixes: #5610 (in part)

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### How Has This Been Tested?

This has been tested in the qa env, as it is only a UI change.

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan ui

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by sureshanaparti** [comment]
@blueorangutan package

**Review by utchoang — APPROVED** [review]
Code LGTM!

File: ui/src/config/section/config.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ export default {
           label: 'label.remove.ldap',
           message: 'message.remove.ldap',
           dataView: true,
-          args: ['hostname'],
+          args: ['hostname', 'port', 'domainid'],
```
Comment (sureshanaparti) [inline ui/src/config/section/config.js:54]: @DaanHoogland Currently, remove LDAP option is provided in the details page, so can these details auto-picked from the LDAP details, instead prompting user to input?

LDAP details =>

LDAP_Details

Old behavior =>

Removed_LDAP_Old

New behavior =>

Removed_LDAP_New

**Comment by DaanHoogland** [comment]
@blueorangutan ui

**Comment by sureshanaparti** [comment]
Verified. LGTM.

**Review by sureshanaparti — APPROVED** [review]

**Pull request merged**